ledger/nomos-ledger/src/mantle/sdp/mod.rs

```diff
@@ -733,7 +733,7 @@ mod tests {
     fn gc_test_config() -> Config {
         let mut params = HashMap::new();
         params.insert(
-            ServiceType::BlendNetwork,
+            ServiceType::DataAvailability,
```

Comment:
Is this intentional?

Reply:
Yes, this method is used by `test_garbage_collection_different_services`. Previously, the test checked if a blend declaration is expired after some time. But now, the test becomes complicated since we integrates PoQ verification. So, I modified the test to check a DA declaration, instead. For that, we also need to modify this `gc_test_config` method.